frontend/public/components/nav/nav-header.tsx

```diff
@@ -55,25 +99,52 @@ const NavHeader_: React.FC<NavHeaderProps & StateProps> = ({ onPerspectiveSelect
     [isPerspectiveDropdownOpen, togglePerspectiveOpen],
   );
 
-  const perspectiveItems = React.useMemo(
-    () =>
-      perspectiveExtensions.map((nextPerspective: Perspective) => (
+  const perspectiveItems = React.useMemo(() => {
+    const items = perspectiveExtensions.map((nextPerspective: Perspective) => (
+      <DropdownItem
+        key={nextPerspective.properties.id}
+        onClick={(event: React.MouseEvent<HTMLLinkElement>) =>
+          onPerspectiveSelect(event, nextPerspective)
+        }
+        isHovered={nextPerspective.properties.id === activePerspective}
+        component="button"
+      >
+        <Title headingLevel="h2" size="md" data-test-id="perspective-switcher-menu-option">
+          <span className="oc-nav-header__icon">{nextPerspective.properties.icon}</span>
+          {nextPerspective.properties.name}
+        </Title>
+      </DropdownItem>
+    ));
+    if (acmLink) {
+      const acmPerspective: Perspective = {
+        type: 'Perspective',
+        properties: {
+          id: ACM_LINK_ID,
+          name: 'Advanced Cluster Management',
+          icon: <CubesIcon />,
+          getLandingPageURL: () => acmLink.spec.href,
+          getK8sLandingPageURL: acmLink.spec.href,
+          getImportRedirectURL: acmLink.spec.href,
+        },
+      };
+      items.push(
         <DropdownItem
-          key={nextPerspective.properties.id}
+          key={acmPerspective.properties.id}
           onClick={(event: React.MouseEvent<HTMLLinkElement>) =>
-            onPerspectiveSelect(event, nextPerspective)
+            onPerspectiveSelect(event, acmPerspective)
           }
-          isHovered={nextPerspective.properties.id === activePerspective}
+          isHovered={acmPerspective.properties.id === activePerspective}
           component="button"
```

Comment:
Is it possible to make this an `a` element so we don't need a click handler and the browser can just treat it as a link?

Reply:
I made it an `a` and the cursor changed to an `I` icon, other than that behaved the same.  I believe the click handler is still needed to handle setting the active perspective and `setPerspectiveDropdownOpen(false)` plus `onPerspectiveSelected()`.  

When the `?perspective=development` query params are in place, we may be able to just append the perspective query params to the URL of a `a` component, but still would need to do the other stuff in the click handler.

Reply:
I don't think you need to call `setPerspectiveDropdownOpen` or `onPerspectiveSelected` for the ACM link since it links away from console.

Reply:
Hi @spadgett, oh yeah, makes sense, duh :-) fixed